task/03-date-tasks.js

```diff
@@ -94,7 +108,19 @@ function timeSpanToString(startDate, endDate) {
  *    Date.UTC(2016,3,5,21, 0) => Math.PI/2
  */
 function angleBetweenClockHands(date) {
-    throw new Error('Not implemented');
+  let time = new Date(date);
+  let angle = 0.5 * (60 * time.getUTCHours() - 11 * time.getUTCMinutes());
+  let radian = Math.PI /180;
+  
+  if(angle > 180)
```

Comment:
it is necessary to observe indents after "if" in this case.
Also left bracket should be on the same line as conditional statement